docs/system/yocto/yocto.md

```diff
@@ -38,6 +38,8 @@ scripts/bygg \
 
 *See [Building with Docker](yocto-build-with-docker.md) for details.*
 
+To get more information about what you can do with it use /script/bygg --help.
```

Comment:
To get more information about what you can do with it, run `script/bygg --help`.